AdobeStockImageAdminUi/etc/config.xml

```diff
@@ -1,16 +0,0 @@
-<?xml version="1.0"?>
-<!--
-/**
- * Copyright © Magento, Inc. All rights reserved.
- * See COPYING.txt for license details.
- */
--->
-<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Store:etc/config.xsd">
```

Comment:
That config.xml change has already been merged to 2.0-develop

Reply:
Not sure I get what you mean. I removed this config from `AdobeStockImageAdminUi` but it is present in `MediaGalleryUi` with `0` value. 
https://github.com/magento/adobe-stock-integration/pull/1611/files#diff-7c5661c33a5e5b316c37db653ae2700cL12

Reply:
I guess this can be marked as resolved after the branch was updated